xml/cl.xml

```diff
@@ -6933,6 +6946,30 @@ server's OpenCL/api-docs repository.
             <require comment="cl_device_info">
                 <enum name="CL_DEVICE_MEMORY_CAPABILITIES_IMG"/>
             </require>
+        </extension>
+         <extension name="cl_img_safety_mechanisms" revision="0.0.0" supported="opencl">
```

Comment:
Do you want this to be version 1.0.0 to match the version in the extension doc?

```suggestion
         <extension name="cl_img_safety_mechanisms" revision="1.0.0" supported="opencl">
```

Reply:
Thanks Ben, I've changed the revision to 1.0.0.